diff -Naur freeradius-server-2.1.7/dialup_admin/bin/backup_radacct freeradius-server-2.1.7-fix-perl-scripts/dialup_admin/bin/backup_radacct
--- freeradius-server-2.1.7/dialup_admin/bin/backup_radacct	2009-09-14 16:43:29.000000000 +0200
+++ freeradius-server-2.1.7-fix-perl-scripts/dialup_admin/bin/backup_radacct	2010-01-09 12:37:31.000000000 +0100
@@ -1,6 +1,6 @@
 #!/usr/bin/perl
 use POSIX;
-use File::Temp;
+use File::Temp qw(tempfile tempdir);
 
 $conf=shift||'/data/local/dialupadmin/conf/admin.conf';
 $back_days = 80;
diff -Naur freeradius-server-2.1.7/dialup_admin/bin/clean_radacct freeradius-server-2.1.7-fix-perl-scripts/dialup_admin/bin/clean_radacct
--- freeradius-server-2.1.7/dialup_admin/bin/clean_radacct	2009-09-14 16:43:29.000000000 +0200
+++ freeradius-server-2.1.7-fix-perl-scripts/dialup_admin/bin/clean_radacct	2010-01-09 12:37:31.000000000 +0100
@@ -5,7 +5,7 @@
 # Works with mysql and postgresql
 #
 use POSIX;
-use File::Temp;
+use File::Temp qw(tempfile tempdir);
 
 $conf=shift||'/usr/local/dialup_admin/conf/admin.conf';
 $back_days = 35;
diff -Naur freeradius-server-2.1.7/dialup_admin/bin/log_badlogins freeradius-server-2.1.7-fix-perl-scripts/dialup_admin/bin/log_badlogins
--- freeradius-server-2.1.7/dialup_admin/bin/log_badlogins	2009-09-14 16:43:29.000000000 +0200
+++ freeradius-server-2.1.7-fix-perl-scripts/dialup_admin/bin/log_badlogins	2010-01-09 12:37:31.000000000 +0100
@@ -14,7 +14,7 @@
 
 use Date::Manip qw(ParseDate UnixDate);
 use Digest::MD5;
-use File::Temp;
+use File::Temp qw(tempfile tempdir);
 $|=1;
 
 $file=shift||'none';
diff -Naur freeradius-server-2.1.7/dialup_admin/bin/monthly_tot_stats freeradius-server-2.1.7-fix-perl-scripts/dialup_admin/bin/monthly_tot_stats
--- freeradius-server-2.1.7/dialup_admin/bin/monthly_tot_stats	2009-09-14 16:43:29.000000000 +0200
+++ freeradius-server-2.1.7-fix-perl-scripts/dialup_admin/bin/monthly_tot_stats	2010-01-09 12:37:31.000000000 +0100
@@ -1,6 +1,6 @@
 #!/usr/bin/perl
 use POSIX;
-use File::Temp;
+use File::Temp qw(tempfile tempdir);
 
 # Log in the mtotacct table aggregated accounting information for
 # each user spaning in one month period.
diff -Naur freeradius-server-2.1.7/dialup_admin/bin/tot_stats freeradius-server-2.1.7-fix-perl-scripts/dialup_admin/bin/tot_stats
--- freeradius-server-2.1.7/dialup_admin/bin/tot_stats	2009-09-14 16:43:29.000000000 +0200
+++ freeradius-server-2.1.7-fix-perl-scripts/dialup_admin/bin/tot_stats	2010-01-09 12:37:31.000000000 +0100
@@ -1,6 +1,6 @@
 #!/usr/bin/perl
 use POSIX;
-use File::Temp;
+use File::Temp qw(tempfile tempdir);
 
 # Log in the totacct table aggregated daily accounting information for
 # each user.
diff -Naur freeradius-server-2.1.7/dialup_admin/bin/truncate_radacct freeradius-server-2.1.7-fix-perl-scripts/dialup_admin/bin/truncate_radacct
--- freeradius-server-2.1.7/dialup_admin/bin/truncate_radacct	2009-09-14 16:43:29.000000000 +0200
+++ freeradius-server-2.1.7-fix-perl-scripts/dialup_admin/bin/truncate_radacct	2010-01-09 12:37:31.000000000 +0100
@@ -5,7 +5,7 @@
 # Works with mysql and postgresql
 #
 use POSIX;
-use File::Temp;
+use File::Temp qw(tempfile tempdir);
 
 $conf=shift||'/usr/local/dialup_admin/conf/admin.conf';
 $back_days = 90;
