--- src/modules/rlm_yubikey/rlm_yubikey.c	2012-09-28 17:41:11.000000000 +0200
+++ src/modules/rlm_yubikey/rlm_yubikey.c.oden	2012-10-03 14:20:21.533549117 +0200
@@ -33,8 +33,8 @@ RCSID("$Id$")
 #include "md5.h"
 
 
-#define KEYS_PATH "main.aeskeys.key_"
-#define USERS_PATH "main.users"
+#define KEYS_PATH "/etc/raddb/yubico/main.aeskeys.key_"
+#define USERS_PATH "/etc/raddb/yubico/main.users"
 
 
 /*
@@ -182,7 +182,7 @@ static int yubikey_auth_core(myConf_t *m
     int passLen = 0, session = 0, counter = 0, i = 0;
     MD5_CTX ctx;
     int result = 0;
-    char *filename = "/usr/local/etc/raddb/yubico/users";
+    char *filename = "/etc/raddb/yubico/users";
 
     //get password by removing the last 32 characters of the password
     if (strlen(request->password->vp_strvalue) <= 32)
