diff -Naur freeradius-server-2.1.6/src/modules/rlm_ruby/rlm_ruby.c freeradius-server-2.1.6-fix-format-errors/src/modules/rlm_ruby/rlm_ruby.c
--- freeradius-server-2.1.6/src/modules/rlm_ruby/rlm_ruby.c	2009-05-18 13:13:55.000000000 +0200
+++ freeradius-server-2.1.6-fix-format-errors/src/modules/rlm_ruby/rlm_ruby.c	2009-06-11 23:02:48.000000000 +0200
@@ -87,7 +87,7 @@
     char *msg;
     status = FIX2INT(msg_type);
     msg = STR2CSTR(rb_msg);
-    radlog(status, msg);
+    radlog(status, "%s", msg);
     return Qnil;
 }
 
