--- src/modules/rlm_ruby/configure.in.ruby	2012-09-10 19:51:34.000000000 +0800
+++ src/modules/rlm_ruby/configure.in	2012-11-05 11:13:30.228531800 +0800
@@ -48,21 +48,21 @@
 
 	if test x$fail = x; then
 		#We use fetch, cause [] are disapearing somewere
-		RB_PREFIX=`${RUBYBIN} -e "require 'rbconfig'; puts Config::CONFIG.fetch('prefix')"`
-		RB_EXEC_PREFIX=`${RUBYBIN} -e "require 'rbconfig'; puts Config::CONFIG.fetch('exec_prefix')"`
+		RB_PREFIX=`${RUBYBIN} -e "require 'rbconfig'; puts RbConfig::CONFIG.fetch('prefix')"`
+		RB_EXEC_PREFIX=`${RUBYBIN} -e "require 'rbconfig'; puts RbConfig::CONFIG.fetch('exec_prefix')"`
 		changequote(<<, >>)dnl
 		RB_VERSION=`${RUBYBIN} -e 'puts RUBY_VERSION'`
 		changequote([, ])dnl
 #		RB_LIBS=`${RUBYBIN} -e "require 'rbconfig'; puts Config::CONFIG.fetch('LIBRUBY')"`
 #		RB_LIB_DIR=`${RUBYBIN} -e "require 'rbconfig'; puts Config::CONFIG.fetch('libdir')"`
 #		RB_EXTRA_LIBS=`${RUBYBIN} -e "require 'rbconfig'; puts Config::CONFIG.fetch('LIBS')"`
-		RB_LIBS=`${RUBYBIN} -e "require 'rbconfig'; puts Config::CONFIG.fetch('LIBRUBYARG_SHARED')"`
-		RB_INC_DIR=`${RUBYBIN} -e "require 'rbconfig'; puts Config::CONFIG.fetch('includedir')"`
-		RB_CFLAGS=`${RUBYBIN} -e "require 'rbconfig'; puts Config::CONFIG.fetch('CFLAGS')"`
-		RB_ARCH_DIR=`${RUBYBIN} -e "require 'rbconfig'; puts Config::CONFIG.fetch('archdir')"`
+		RB_LIBS=`${RUBYBIN} -e "require 'rbconfig'; puts RbConfig::CONFIG.fetch('LIBRUBYARG_SHARED')"`
+		RB_INC_DIR=`${RUBYBIN} -e "require 'rbconfig'; puts RbConfig::CONFIG.fetch('includedir')"`
+		RB_CFLAGS=`${RUBYBIN} -e "require 'rbconfig'; puts RbConfig::CONFIG.fetch('CFLAGS')"`
+		RB_ARCH_DIR=`${RUBYBIN} -e "require 'rbconfig'; puts RbConfig::CONFIG.fetch('arch')"`
 
 		old_CFLAGS=$CFLAGS
-		CFLAGS="$CFLAGS $RB_CFLAGS -I${RB_ARCH_DIR} -I${RB_INC_DIR}"
+		CFLAGS="$CFLAGS $RB_CFLAGS -I${RB_INC_DIR}/${RB_ARCH_DIR} -I${RB_INC_DIR}"
 #		smart_try_dir=$RB_INC_DIR
 		FR_SMART_CHECK_INCLUDE(ruby.h)
 		if test "x$ac_cv_header_ruby_h" != "xyes"; then
