diff -Naur freeradius-server-2.2.3/src/modules/rlm_sql/drivers/rules.mak freeradius-server-2.2.3.oden/src/modules/rlm_sql/drivers/rules.mak
--- freeradius-server-2.2.3/src/modules/rlm_sql/drivers/rules.mak	2013-12-11 21:10:12.000000000 +0100
+++ freeradius-server-2.2.3.oden/src/modules/rlm_sql/drivers/rules.mak	2014-02-18 14:28:30.659614040 +0100
@@ -101,7 +101,7 @@
 	done
 
 $(TARGET).la: $(LT_OBJS)
-	$(LIBTOOL) --mode=link --tag=CC $(CC) -release $(RADIUSD_VERSION_STRING) \
+	$(LIBTOOL) --mode=link --tag=CC $(CC) -avoid-version \
 	-module $(LINK_MODE) $(LDFLAGS) $(RLM_SQL_LDFLAGS) -o $@ \
 	-rpath $(libdir) $^ $(RLM_SQL_LIBS)
 
diff -Naur freeradius-server-2.2.3/src/modules/rules.mak freeradius-server-2.2.3.oden/src/modules/rules.mak
--- freeradius-server-2.2.3/src/modules/rules.mak	2013-12-11 21:10:12.000000000 +0100
+++ freeradius-server-2.2.3.oden/src/modules/rules.mak	2014-02-18 14:27:41.456611289 +0100
@@ -120,7 +120,7 @@
 	done
 
 $(TARGET).la: $(RLM_SUBDIRS) $(LT_OBJS)
-	$(LIBTOOL) --mode=link --tag=CC $(CC) -release $(RADIUSD_VERSION_STRING) \
+	$(LIBTOOL) --mode=link --tag=CC $(CC) -avoid-version \
 	-module $(LINK_MODE) $(LDFLAGS) $(RLM_LDFLAGS) -o $@     \
 	-rpath $(libdir) $^ $(LIBRADIUS) $(RLM_LIBS) $(LIBS)
 
