diff -Naurp freeradius-server-2.2.3/src/modules/rlm_krb5/rlm_krb5.c freeradius-server-2.2.3.oden/src/modules/rlm_krb5/rlm_krb5.c
--- freeradius-server-2.2.3/src/modules/rlm_krb5/rlm_krb5.c	2013-12-11 21:10:12.000000000 +0100
+++ freeradius-server-2.2.3.oden/src/modules/rlm_krb5/rlm_krb5.c	2014-02-18 14:25:34.076604166 +0100
@@ -30,7 +30,7 @@ RCSID("$Id: 42132624367187730c30565a43f1
 
 /* krb5 includes */
 #include <krb5.h>
-#include <com_err.h>
+#include <et/com_err.h>
 
  /* Arbitrary 64char limit on service names */
 #define SERVICE_NAME_LEN 64
