diff -Naur freeradius-server-2.1.12/src/modules/rlm_perl/configure freeradius-server-2.1.12.oden/src/modules/rlm_perl/configure
--- freeradius-server-2.1.12/src/modules/rlm_perl/configure	2011-09-30 16:12:07.000000000 +0200
+++ freeradius-server-2.1.12.oden/src/modules/rlm_perl/configure	2011-11-12 16:02:35.246868123 +0100
@@ -3249,7 +3249,7 @@
 	  fi
 
 	  old_LIBS="$LIBS"
-	  LIBS="$old_LIBS `perl -MExtUtils::Embed -e ldopts`"
+	  LIBS="$old_LIBS `perl -MExtUtils::Embed -e ldopts|perl -pi -e 's|-Wl\,--no-undefined||g'`"
 	  cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
@@ -3295,7 +3295,7 @@
       conftest$ac_exeext conftest.$ac_ext
 
 	  if test "x$BROKEN" = "x"; then
-        	perl_ldflags='`perl -MExtUtils::Embed -e ldopts`'
+        	perl_ldflags="`perl -MExtUtils::Embed -e ldopts|perl -pi -e 's|-Wl\,--no-undefined||g'`"
 	  else
 		fail="$fail libperl.so"
 		targetname=
diff -Naur freeradius-server-2.1.12/src/modules/rlm_perl/configure.in freeradius-server-2.1.12.oden/src/modules/rlm_perl/configure.in
--- freeradius-server-2.1.12/src/modules/rlm_perl/configure.in	2011-09-30 16:12:07.000000000 +0200
+++ freeradius-server-2.1.12.oden/src/modules/rlm_perl/configure.in	2011-11-12 16:01:29.073181299 +0100
@@ -37,14 +37,14 @@
 	  fi
 
 	  old_LIBS="$LIBS"
-	  LIBS="$old_LIBS `perl -MExtUtils::Embed -e ldopts`"
+	  LIBS="$old_LIBS `perl -MExtUtils::Embed -e ldopts|perl -pi -e 's|-Wl\,--no-undefined||g'`"
 	  AC_TRY_LINK([extern char boot_DynaLoader();],
 		      [ boot_DynaLoader()],
 		      BROKEN=,
 		      BROKEN="yes")
 
 	  if test "x$BROKEN" = "x"; then
-        	perl_ldflags='`perl -MExtUtils::Embed -e ldopts`'
+        	perl_ldflags="`perl -MExtUtils::Embed -e ldopts|perl -pi -e 's|-Wl\,--no-undefined||g'`"
 	  else
 		fail="$fail libperl.so"
 		targetname=
